std/json.d

```diff
@@ -34,28 +52,9 @@ Synopsis:
     // list already exists, so .object optional
     jj["list"].array ~= JSONValue("D");
 
-    s = j.toString();
-    writeln(s);
-----
-
-Copyright: Copyright Jeremie Pelletier 2008 - 2009.
-License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
-Authors:   Jeremie Pelletier, David Herberth
-References: $(LINK http://json.org/)
-Source:    $(PHOBOSSRC std/_json.d)
-*/
-/*
-         Copyright Jeremie Pelletier 2008 - 2009.
-Distributed under the Boost Software License, Version 1.0.
-   (See accompanying file LICENSE_1_0.txt or copy at
-         http://www.boost.org/LICENSE_1_0.txt)
-*/
-module std.json;
-
-import std.conv;
-import std.range.primitives;
-import std.array;
-import std.traits;
+    string jjStr = `{"language":"D","list":["a","b","c","D"],"rating":3.14}`;
+    assert(jj.toString == jjStr);
```

Comment:
Ugh. I hate how the example needs to use assert instead of showing how you would output to writeln.

This has the unsatisfying result of Mr. Novice copying the code into a file, running it, and having zero feedback of anything. I'm going to write a forum post about this.